aea/skills/error/__init__.py

```diff
@@ -18,6 +18,3 @@
 # ------------------------------------------------------------------------------
 
 """This module contains the implementation of the error skill."""
-from aea.configurations.base import PublicId
```

Comment:
this caused issues

Reply:
It is also not particularly neat, as the constant easily can get out of line with the config file.

Reply:
Ideally, the public id would be constructed from the config file.

Reply:
yes, it is how it works now. The public id is retrieved from the configuration object.